### PR TITLE
Created LICENSE

### LINKED ISSUE #1: [Automatic validation] Missing License
# How to fix
Add a license for this repository. See [help](https://help.github.com/en/articles/licensing-a-repository) for guidance. Private repositories don't need a license.
# About validation
These issues are created, closed and reopened by [repository validator](https://github.com/protacon/repository-validator) when commits are pushed to repository. 

If there are problems, please add an issue to [repository validator](https://github.com/protacon/repository-validator)

DO NOT change the name of this issue. Names are used to identify the issues created by automation.

### PR DESCRIPTION
**Pull request by Hekku2**
This public repository is missing license. Would MIT be a good choice here?

Closes #1

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Protacon
```
Comment (aeosys) [inline LICENSE:3]: Should this be Protacon Group Ltd 🤔

**Review by aeosys — APPROVED** [review]
Started to look into this yesterday but got stuck reading about how to use years in copyright and thinking what company should actually be placed in it. There seems to be all kind of usages. Though it really does not matter with this repo. 😄

File: LICENSE (hunk 1 above)
Comment (Hekku2) [reply LICENSE:3]: Hmm, I'll try to verify this.

Comment (Hekku2) [reply LICENSE:3]: I verified this from Pirita, @ 2019 Protacon is fine

**Pull request merged**